README.md
# NAME

Task::Badge::Depot - All Badge::Depot badges

<div>
    <p>
    <img src="https://img.shields.io/badge/perl-5.10+-blue.svg" alt="Requires Perl 5.10+" />
    <a href="https://travis-ci.org/Csson/p5-Task-Badge-Depot"><img src="https://api.travis-ci.org/Csson/p5-Task-Badge-Depot.svg?branch=master" alt="Travis status" /></a>
    <a href="http://cpants.cpanauthors.org/dist/Task-Badge-Depot-0.0102"><img src="https://badgedepot.code301.com/badge/kwalitee/Task-Badge-Depot/0.0102" alt="Distribution kwalitee" /></a>
    <a href="http://matrix.cpantesters.org/?dist=Task-Badge-Depot%200.0102"><img src="https://badgedepot.code301.com/badge/cpantesters/Task-Badge-Depot/0.0102" alt="CPAN Testers result" /></a>
    </p>
</div>

# VERSION

Version 0.0102, released 2016-02-22.

# TASK CONTENTS

## Badge::Depot, with badges

### [Badge::Depot](https://metacpan.org/pod/Badge::Depot) 0.0104

### [Badge::Depot::Plugin::Coverage](https://metacpan.org/pod/Badge::Depot::Plugin::Coverage) 0.0101

### [Badge::Depot::Plugin::Cpantesters](https://metacpan.org/pod/Badge::Depot::Plugin::Cpantesters) 0.0101

### [Badge::Depot::Plugin::Gratipay](https://metacpan.org/pod/Badge::Depot::Plugin::Gratipay) 0.0102

### [Badge::Depot::Plugin::Kwalitee](https://metacpan.org/pod/Badge::Depot::Plugin::Kwalitee) 0.0101

### [Badge::Depot::Plugin::Perl](https://metacpan.org/pod/Badge::Depot::Plugin::Perl) 0.0103

### [Badge::Depot::Plugin::Travis](https://metacpan.org/pod/Badge::Depot::Plugin::Travis) 0.0202

# SOURCE

[https://github.com/Csson/p5-Task-Badge-Depot](https://github.com/Csson/p5-Task-Badge-Depot)

# HOMEPAGE

[https://metacpan.org/release/Task-Badge-Depot](https://metacpan.org/release/Task-Badge-Depot)

# AUTHOR

Erik Carlsson <info@code301.com>

# COPYRIGHT AND LICENSE

This software is copyright (c) 2016 by Erik Carlsson.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.


Changes
Revision history for Task-Badge-Depot

{{$NEXT}}
 [API Changes]

 [Bug Fixes]

 [Documentation]

 [Enhancements]

 [New Features]

 [Other Changes]

 [Requirements]

 [Stats & Dependency Changes]

 [Testing]

0.0102 2016-02-22T15:45:23Z UTC
 [Badge Changes]
 - Add ::Cpantesters and ::Kwalitee badges.
 - Require latest versions on everything.

 [Stats & Dependency Changes]
 - 2 files changed, 21 insertions(+), 7 deletions(-)
 - (run req) + Badge::Depot::Plugin::Cpantesters 0.0101
 - (run req) + Badge::Depot::Plugin::Kwalitee 0.0101
 - (run req) ~ Badge::Depot (any) --> 0.0104
 - (run req) ~ Badge::Depot::Plugin::Coverage (any) --> 0.0101
 - (run req) ~ Badge::Depot::Plugin::Gratipay (any) --> 0.0102
 - (run req) ~ Badge::Depot::Plugin::Perl (any) --> 0.0103
 - (run req) ~ Badge::Depot::Plugin::Travis (any) --> 0.0202
 - (run req) ~ perl 5.010001 --> 5.010000
 - (test req) + IO::Handle (any)
 - (test req) + IPC::Open3 (any)
 - (dev req) + Badge::Depot 0.0103
 - (dev req) + Badge::Depot::Plugin::Coverage (any)
 - (dev req) + Badge::Depot::Plugin::Cpantesters (any)
 - (dev req) + Badge::Depot::Plugin::Kwalitee (any)
 - (dev req) + Badge::Depot::Plugin::Perl (any)
 - (dev req) + Badge::Depot::Plugin::Travis (any)
 - (dev req) + Dist::Zilla 5.000
 - (dev req) + Dist::Zilla::Plugin::Authority 1.009
 - (dev req) + Dist::Zilla::Plugin::BumpVersionAfterRelease::Transitional
   0.007
 - (dev req) + Dist::Zilla::Plugin::ChangeStats::Dependencies::Git 0.0100
 - (dev req) + Dist::Zilla::Plugin::Git 2.036
 - (dev req) + Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch 0.013
 - (dev req) + Dist::Zilla::Plugin::Git::Commit (any)
 - (dev req) + Dist::Zilla::Plugin::NextRelease::Grouped 0.0200
 - (dev req) + Dist::Zilla::Plugin::RewriteVersion::Transitional 0.007
 - (dev req) + Dist::Zilla::Plugin::Test::Compile 2.054
 - (dev req) + Dist::Zilla::Plugin::Test::EOF 0.0501
 - (dev req) + Dist::Zilla::Plugin::Test::EOL 0.18
 - (dev req) + Dist::Zilla::Plugin::Test::Kwalitee::Extra v0.2.1
 - (dev req) + Dist::Zilla::Plugin::Test::NoTabs 0.15
 - (dev req) + Dist::Zilla::Plugin::Test::Version 1.05
 - (dev req) + Pod::Weaver::Section::GenerateSection 1.01
 - (dev req) + Test::EOF (any)
 - (dev req) + Test::EOL (any)
 - (dev req) + Test::More 0.88
 - (dev req) + Test::NoTabs 1.4
 - (dev req) + Test::Version 1
 - (dev req) ~ Dist::Zilla::Plugin::CheckChangesHasContent (any) --> 0.008
 - (dev req) ~ Dist::Zilla::Plugin::Clean (any) --> 0.07
 - (dev req) ~ Dist::Zilla::Plugin::CopyFilesFromBuild (any) --> 0.151680
 - (dev req) ~ Dist::Zilla::Plugin::Git::Contributors (any) --> 0.017
 - (dev req) ~ Dist::Zilla::Plugin::GithubMeta (any) --> 0.54
 - (dev req) ~ Dist::Zilla::Plugin::InstallRelease (any) --> 0.008
 - (dev req) ~ Dist::Zilla::Plugin::MetaProvides::Package (any) -->
   2.003001
 - (dev req) ~ Dist::Zilla::Plugin::ReadmeAnyFromPod (any) --> 0.150250
 - (dev req) ~ Dist::Zilla::Plugin::RunExtraTests (any) --> 0.028
 - (dev req) ~ Dist::Zilla::Plugin::TaskWeaver (any) --> 0.101628
 - (dev req) ~ Dist::Zilla::Plugin::Test::ReportPrereqs (any) --> 0.021
 - (dev req) ~ Dist::Zilla::Plugin::TravisYML (any) --> 1.15
 - (dev req) ~ Pod::Weaver::Section::Badges (any) --> 0.0401
 - (dev req) ~ Test::Warnings (any) --> 0.026
 - (dev req) - Dist::Zilla::Plugin::CheckExtraTests
 - (dev req) - Dist::Zilla::Plugin::MetaProvides::Class
 - (dev req) - Dist::Zilla::Plugin::NextRelease
 - (dev req) - Dist::Zilla::Plugin::NextVersion::Semantic
 - (dev req) - Dist::Zilla::Plugin::OurPkgVersion
 - (dev req) - Dist::Zilla::Plugin::PodnameFromClassname
 - (dev req) - Dist::Zilla::Plugin::Prereqs::Plugins
 - (dev req) - Dist::Zilla::Plugin::PreviousVersion::Changelog
 - (dev req) - Dist::Zilla::Plugin::ReversionOnRelease
 - (dev req) - Pod::Weaver::Section::Homepage::DefaultCPAN
 - (dev req) - Pod::Weaver::Section::Source::DefaultGitHub
 - (dev sug) + Dist::Iller 0.1407
 - (dev sug) + Dist::Iller::Config::Author::CSSON 0.0309

0.0101 2016-01-19T20:26:11Z UTC
 [Enhancements]
 - Added coverage badge.

0.0100 2015-03-14T23:18:06Z CET
 - Initial release


Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.043.
use strict;
use warnings;

use 5.010000;

use ExtUtils::MakeMaker;

my %WriteMakefileArgs = (
  "ABSTRACT" => "All Badge::Depot badges",
  "AUTHOR" => "Erik Carlsson <info\@code301.com>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "Task-Badge-Depot",
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.010000",
  "NAME" => "Task::Badge::Depot",
  "PREREQ_PM" => {
    "Badge::Depot" => "0.0104",
    "Badge::Depot::Plugin::Coverage" => "0.0101",
    "Badge::Depot::Plugin::Cpantesters" => "0.0101",
    "Badge::Depot::Plugin::Gratipay" => "0.0102",
    "Badge::Depot::Plugin::Kwalitee" => "0.0101",
    "Badge::Depot::Plugin::Perl" => "0.0103",
    "Badge::Depot::Plugin::Travis" => "0.0202"
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.96"
  },
  "VERSION" => "0.0103",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "Badge::Depot" => "0.0104",
  "Badge::Depot::Plugin::Coverage" => "0.0101",
  "Badge::Depot::Plugin::Cpantesters" => "0.0101",
  "Badge::Depot::Plugin::Gratipay" => "0.0102",
  "Badge::Depot::Plugin::Kwalitee" => "0.0101",
  "Badge::Depot::Plugin::Perl" => "0.0103",
  "Badge::Depot::Plugin::Travis" => "0.0202",
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Test::More" => "0.96"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);
